### PR TITLE
Update to latest sDDF

### PR DESCRIPTION
**Pull request by Kswin01**
Update the sDDF hash, and modify the kitty system appropriately.

### CODE REVIEW

**Comment by Ivan-Velickovic** [comment]
In the commit message for the commit that removes the ARP component, can you add a line saying that we don't need the ARP component is not necessary in the case of the Kitty example because our clients use lwIP which can handle ARP requests on its own.

File: examples/kitty/Makefile [hunk 1]
```diff
@@ -195,6 +197,7 @@ NETWORK_COMPONENTS_INCLUDE := \
 	-Isrc/ethernet_config
 NETWORK_COMPONENT_CFLAGS := \
 	$(NETWORK_COMPONENTS_INCLUDE) \
+	-DNUM_NETWORK_CLIENTS=2 \
```
Comment (Ivan-Velickovic) [inline examples/kitty/Makefile:200]: I thought the ethernet config file already had this?

File: examples/kitty/src/ethernet_config/ethernet_config.h [hunk 2]
```diff
@@ -2,9 +2,8 @@
 
 #include <sddf/network/queue.h>
 
-#define NUM_CLIENTS 3
+#define NUM_CLIENTS 2
```
Comment (Ivan-Velickovic) [inline examples/kitty/src/ethernet_config/ethernet_config.h:5]: This should probably be renamed to `NETWORK_NUM_CLIENTS` for the latest sDDF change I think.

File: examples/webserver/Makefile [hunk 3]
```diff
@@ -171,6 +174,9 @@ NETWORK_COMPONENT_CFLAGS := \
 NETWORK_COMPONENTS_LDFLAGS := -L$(BOARD_DIR)/lib -L$(SDDF)/lib -L$(LIBGCC)
 NETWORK_COMPONENTS_LIBS := -lmicrokit -Tmicrokit.ld -lc -lgcc $(BUILD_DIR)/sddf_util_cache.o $(BUILD_DIR)/sddf_util_printf.o
 
+NETWORK_COMPONENTS_LDFLAGS := -L$(BOARD_DIR)/lib -L$(LIBGCC) -L$(LIBC)
+NETWORK_COMPONENTS_LIBS := -lmicrokit -Tmicrokit.ld -lc -lgcc $(BUILD_DIR)/sddf_util_cache.o $(BUILD_DIR)/sddf_util_printf.o
+
```
Comment (Ivan-Velickovic) [inline examples/webserver/Makefile:179]: These are getting defined twice?

Comment (Ivan-Velickovic) [reply examples/webserver/Makefile:179]: Lines 174 and 175 already seem to define them

Comment (Kswin01) [reply examples/webserver/Makefile]: Also removed libgcc dependency here.

**Review by Ivan-Velickovic — APPROVED** [review]

**Pull request merged**